src/Expose/main.tex

```diff
@@ -43,28 +41,65 @@
 \setlength\epigraphwidth{.8\textwidth}
 \setlength\epigraphrule{0pt}
 
+\linespread{1.25}
 
 \begin{document}
     % Title page
-	\thispagestyle{fancy}
 	\maketitle
-	\clearpage
 	
 	\section{Einführung}
+		In der Arbeit soll die Frage, in wie weit Endanwender Open Source Software wahrnehmen, untersucht werden. Dabei soll nicht nur die \emph{direkte} Verwendung von Open Source Software, wie es zum Beispiel bei LibreOffice der Fall ist, eine Rolle spielen, sondern auch die \emph{indirekter} Nutzung als Komponente kommerzieller Lösungen wie Chromium bzw. WebKit als Basis von Google Chrome\footcite{is:open:source:right:for:you} aber auch als Werkzeug wie Linux als Betriebssystem für Server\footcite{report:BaselineScenario}.
+		
+		Es wird erwartet, dass viele Nutzer unbewusst bzw. indirekt mit Open Source Software in Kontakt stehen, sie diese also als Teil größerer, kommerzieller Closed Source Software nutzen.
+		
+		Um zuverlässige Daten, die die aktuelle Wahrnehmung der Endanwender widerspiegeln, zu bekommen, wird eine Umfrage durchgeführt. Für die Sicherstellung der Repräsentativität der Umfrage sind mindestens 100 Antworten gefordert. %TODO: Verb 
+		Die Umfrage wird quantitativ mit geschlossenen Fragen gestaltet, um den Aufwand für die Befragten zu minimieren und damit die Teilnahmerate und Datenmenge zu maximieren.\footcite{handbuch:methoden}
+	
+	\section{Umfrage}
+		Zweck der Umfrage ist es einen repräsentativen Überblick über das Nutzerverhalten von Open Source Software nach demographischen Gruppen aufgeschlüsselt zu erhalten. Dabei soll primär Aufschluss über die Verteilung von indirekter zu direkter Nutzung von Open Source Software sowie der Kenntnisstand der Anwender über die Nutzung dieser gegeben werden. In diesem Kontext ist interessant, ob solche Anwendungen primär genutzt werden, weil sie Open Source sind oder diese aufgrund ihrer Qualität und ihrem Bekanntheitsgrad in Gebrauch sind. Ein weiterer Aspekt ist der Vergleich von der Sicht des Nutzers und der von Unternehmen auf Basis einer Schweizer Studie aus der Perspektive von Unternehmen\footcite{oss:studie}.\\
```

Comment:
`ob solche Anwendungen primär genutzt werden` should be `ob solch offene Anwendungen [...]`